src/test/scala/com/amazon/deequ/analyzers/AnalyzerTests.scala

```diff
@@ -847,6 +847,17 @@ class AnalyzerTests extends AnyWordSpec with Matchers with SparkContextSpec with
       analyzer.calculate(df).value shouldBe Success(2.0 / 8.0)
       assert(analyzer.calculate(df).fullColumn.isDefined)
     }
+
+    "compute ratio of sums correctly for numeric data" in withSparkSession { session =>
```

Comment:
@scott-gunn, is there an assumption that all numbers are positive? What happens if denominator equals to 0?

Reply:
There is no assumptions on positive values or zero values. If the denominator were zero, the returned value would be Infinity (Double.PositiveInfinity or Double.NegativeInfinity), it does not throw an exception. I think this is in line with Scala conventions and is a good result. This would be a good unit test to add - so I will add one.

Reply:
Maybe we should also add it in javadocs?

Reply:
Good idea, I have added some extra info to the class docs